scipy/stats/_distribution_infrastructure.py

```diff
@@ -2501,11 +2499,11 @@ def _cdf_complement(self, x, **params):
         return 1 - self._ccdf_dispatch(x, **params)
 
     def _cdf_complement_safe(self, x, **params):
-        out = self._cdf_complement(x, **params)
+        ccdf = self._ccdf_dispatch(x, **params)
+        out = 1 - ccdf
```

Comment:
I could have left this alone and computed `1 - out` below, but that didn't seem as clear to me.